### PR TITLE
Public record ext identif fix

### PR DESCRIPTION
**Pull request by hexplus**

### CODE REVIEW

**Comment by hexplus** [comment]
@amontenegro The public record small fix we talked about.

File: orcid-web/src/main/resources/freemarker/public_profile_v3.ftl [hunk 1]
```diff
@@ -306,7 +306,7 @@
 			        					<div ng-if="showSources['emails']" class="source-line separator" ng-cloak>				                		
 					                		<p>${springMacroRequestContext.getMessage("public_record.sources")}:<br />
 					                			<#list publicGroupedEmails[email] as emailSource>					                																	
-													<#if (emailSource.source)?? && (emailSource.source.sourceName)??>${emailSource.source.sourceName.content}<#else>${springMacroRequestContext.getMessage("manage.liprivate")}</#if> <#if (emailSource.createdDate)??>(${(emailSource.createdDate.value?datetime("yyyy-MM-dd")?date!)})</#if>
+													<#if (emailSource.source)?? && (emailSource.source.sourceName)??>${emailSource.source.sourceName}<#else>${springMacroRequestContext.getMessage("manage.liprivate")}</#if> <#if (emailSource.createdDate)??>(${(emailSource.createdDate.value?datetime("yyyy-MM-dd")?date!)})</#if>
```
Comment (amontenegro) [inline orcid-web/src/main/resources/freemarker/public_profile_v3.ftl]: Hey @hexplus I think this is an error, why you removed the .content from the email source? The source element have the actual name inside .content

**Comment by hexplus** [comment]
@amontenegro Something is happening with my local environment, some changes do not refresh.

**Comment by amontenegro** [comment]
https://trello.com/c/Rac5ed1Z/2902-v2-modal-if-the-name-is-private-the-public-record-doesn-t-entirely-load

**Pull request merged**